integration/env/aws.go

```diff
@@ -129,3 +137,24 @@ func HostAWSAccessKeyToken() string {
 func IDRSAPub() string {
 	return idRSAPub
 }
+
+func NetworkCIDR() string {
+	return fmt.Sprintf("10.12.%d.0/24", subnetBlock())
+}
```

Comment:
I don't think we need to have subnets unique per test. The tenant cluster subnets need to be unique in a host VPC. We create separate host VPC per test. So we need to have subnets unique per tenant cluster in a single test. So in this test you'll need 4 different subnets. E.g. `10.12.0.0/24`, `10.12.1.0/24`, `10.12.2.0./24` and `10.12.3.0/24`.

Reply:
I think this is ok. There does not speak anything against it. This is logically aligned with Azure.

Reply:
I think giving a notion that it doesn't have to be unique is good. Otherwise we can try to solve problems we don't have in the future being confused by the code here.

Also when you change third octet of `/24` network how are you going to split it between tenant clusters?

Reply:
I cannot tell. When this is a problem we should sit together and figure out how it should be.

Reply:
OK, let's do. I'm free to talk after releng sync.